cmd/gaia/sims.mk

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/make -f
+
+########################################
+### Simulations
+
+sim-gaia-nondeterminism:
```

Comment:
There aren't meant to be actually executable atm, correct?

```
cd cmd/gaia; make sim-gaia-fast

Running quick Gaia simulation. This may take several minutes...
can't load package: package github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaia/app: unknown import path "github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaia/app": cannot find package
make: *** [sim-gaia-fast] Error 1
```

Reply:
No, not quite yet. I've dropped them there as prep work for next iteration.

Reply:
I tried to fix them anyway